ui/src/app/pages/workspace/workspace-list.tsx

```diff
@@ -72,6 +80,11 @@ export class WorkspaceList extends React.Component<WithSpinnerOverlayProps, Stat
     }
   }
 
+  canAccessCTWorkspace(tierShortName: string) {
```

Comment:
there's a new function `hasTierAccess(profile: Profile, shortName)` in access-utils that we can use instead